docs/pallas/tpu/pipelining.md

```diff
@@ -208,3 +312,26 @@ simultaneously on each TensorCore. Pallas will handle splitting up the grid
 automatically.
 
 > Note that Megacore is only currently available on TPU `v4` and TPU `v5p`. Supplying `dimension_semantics` annotations is a no-op on other platforms, but *not* specifying it will result in only one TensorCore being used (even if there are more than one available).
+
+When using `pltpu.emit_pipeline`, either `core_axis` should be passed into the function. `core_axis` should be the index of a parallel grid axis to partition the grid on. For example, the following template can be used to partition the kernel over a leading parallel grid dimension:
```

Comment:
"either `core_axis` should be passed into the function" or what? I'm assuming you're mentioning core_axis_name as well?

Reply:
Oops yea. I'll leave out core_axis_name for now since we don't have core_map documented yet.